lib/verdeps/find_package_import_comment_test.go

```diff
@@ -0,0 +1,80 @@
+package verdeps
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestFindPackageImportComment(t *testing.T) {
```

Comment:
What is a package import comment tho and why doesn't /* */ qualify? Is it explicitly after a package statement?